_data/navigation.yaml

```diff
@@ -4,3 +4,10 @@ agile:
     url: /agile/
   - name: More 18F guides
     url: https://18f.gsa.gov/guides/
+accessbility:
+  - name: Home
+    url: /accessbility/
+  - name: Other .Gov Accessibility Sites
+    url: /accessibility/gov-resources
+  - name: More 18F guides
+    url: https://18f.gsa.gov/guides/
```

Comment:
This seems not to be working for me. I cannot see the other two items in the top nav. Am I missing something?

Reply:
I think it's just because of a typo - the key here is missing an "i" before the "b" in accessibility. i.e. `accessibility:`

(Same for the URL for home)